kahuna/public/js/preview/image.html

```diff
@@ -46,6 +46,19 @@
 
     <div class="preview__info" ng:if="! ctrl.hideInfo">
         <div class="flex-container">
+            <ul class="bottom-bar__meta-item preview__collections">
+                <li class="preview__collections__collection"
+                    ng:repeat="collection in ctrl.image.data.collections"
+                    gr-tooltip="Click to open collection: {{collection.data.path.join(' ▸ ')}}"
+                    gr-tooltip-position="top">
+                    <a ui:sref="search.results({query: '~'+collection.data.pathId})"
+                       style="background-color: {{collection.data.cssColour}}"}
```

Comment:
[Now it just leaves it blank](https://github.com/guardian/grid/pull/1631/files#diff-4c4a6be0bd67994ff7b2c4d740e00078R71).